src/main/kotlin/org/sertain/path/PathController.kt

```diff
@@ -0,0 +1,46 @@
+package org.sertain.path
+
+import jaci.pathfinder.Pathfinder
+import jaci.pathfinder.Trajectory
+import jaci.pathfinder.Waypoint
+import jaci.pathfinder.followers.EncoderFollower
+import jaci.pathfinder.modifiers.TankModifier
+
+object PathController {
+    val pathCoordinates = ArrayList<Triple<Double, Double, Double>>()
+
+    var dt = 0.05
```

Comment:
`dt` is `deltaTime`, just FYI. Probably better to be verbose here and use the long name here.